zipline/finance/performance/position_tracker.py

```diff
@@ -350,14 +350,16 @@ def get_positions(self):
                     pass
                 continue
 
-            # Note that this will create a position if we don't currently have
-            # an entry
-            position = positions[sid]
+            position = zp.Position(sid)
```

Comment:
if there are, I'd argue they should be defined in the Position class.

This is also a nice definition of what the exact fields are that we want to make visible to the end user.